### ISSUE TITLE
Fix exit code?

### ISSUE THREAD

**Issue by rlex** [puppetlabs/r10k#254]
If no puppetfile present, r10k puppetfile install still returns 0.
IMHO, this should return non-zero exit code since it can't deploy modules.

```
lex@lancaster > r10k puppetfile install
lex@lancaster > echo $?
0
lex@lancaster > r10k puppetfile install -v
lex@lancaster > echo $?
0
```

**Comment by adrienthebo** [comment]
That looks like a regression in 1.4.0, I should have a fix for this in the
1.4.1 release.

**Comment by rlex** [comment]
Not only in 1.4.0, btw

```
lex@puppet > gem list | grep r10k
r10k (1.3.4)
lex@puppet > r10k puppetfile install -v
[R10K::Task::Puppetfile::Sync - INFO] Loading modules from Puppetfile into queue
lex@puppet > echo $?
0
```

**Comment by adrienthebo** [comment]
I suppose that this is one of the few times I wish that I actually introduced a regression instead of a longer standing issue. 1.4.0 made a number of improvements that make it a lot easier to fix in the 1.4.x series, so unless there's a very compelling reason to fix this in 1.3.x as well then this will be just going into 1.4.x.

**Comment by adrienthebo** [comment]
Merged into 1.4.x in a07bae0, this should be released in 1.4.1.
